### ISSUE TITLE
[Bug] autocomplete specs does not show with sudo

### ISSUE THREAD

**Issue by dipakparmar** [withfig/autocomplete#176]
### Completion Spec

What completion spec does this bug refer to? Or is it Fig itself?

**_Fig itself_**

### Description

Describe the issue that you're seeing.

**_No autocomplete_**

### Steps to reproduce

Clear steps describing how to reproduce the issue. **Please** link to a reproduction, this makes your issue _much_ easier to diagnose (seriously).

**_Use <kbd>sudo</kbd> before any command_**

### Expected result

What should happen?

**_Should ignore sudo or change permission to run the script and then ignore sudo and run autocomplete specs with sudo permission_**

### Actual result

What happened.

**_No autocomplete spec_**

### System Information

*Fig Version:*  1.0.40 (B188)

*OS:* macOS Big Sur 11.3

*Device:* macBook

### Potential Solution

(optional) What could be a solution to solve this bug?

**_Should ignore sudo or change permission to run the script and then ignore sudo and run autocomplete specs with sudo permission_**

**Comment by mattschrage** [comment]
Thanks for reporting @dipakparmar! So the issue here is that Fig is looking at the tokenized edit buffer `["sudo", ...]`, trying to load a completion spec for `sudo` and then not finding one.

I think the fix here is pretty simple. Create a `sudo` completion spec and make sure to add an argument that has the `isCommand` property set.
```
args: {
  name: "command",
  description: "command to run with elevated permissions"
  isCommand: true
}
```

**Comment by CastaChick** [comment]
Hi, May I try to work on this issue?

**Comment by dipakparmar** [comment]
@mattschrage this seems fixed on the latest update `v1.0.41` 

Thank you! 😀
